src/notation/view/widgets/ChordSymbolStyleEditor.qml

```diff
@@ -0,0 +1,148 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-CLA-applies
+ *
+ * MuseScore
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2021 MuseScore BVBA and others
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+import QtQuick 2.0
+import QtQuick.Controls 2.0
+import QtQuick.Layouts 1.3
+
+import MuseScore.UiComponents 1.0
+import MuseScore.Ui 1.0
+import MuseScore.NotationScene 1.0
+
+Rectangle {
+    id: root
+    color: ui.theme.backgroundPrimaryColor
+
+    ChordSymbolEditorModel {
+        id: chordSymbolEditorModel
+    }
+
+    height: grid.height
+
+    Component {
+        id: chordStyleDelegate
+
+        FlatButton {
+            id: button
+
+            width: grid.cellWidth-5
+            height: grid.cellHeight-5
+
+            Text {
+                id: buttontext
+                text: styleName
```

Comment:
When you want to display text on a button, it is better to use the `text:` property of FlatButton. 
Also, it is preferred to use our own components (StyledTextLabel) instead of QML's default ones, because our own ones have the correct styling.

Reply:
Ok. I will change it finally(if it is still there). Because anyway I will be changing it to a canvas or smth.